app/stac_api/serializers.py

```diff
@@ -85,6 +89,75 @@ def data(self):
         return ReturnDict(ret, serializer=self)
 
 
+class LandingPageLinkSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = LandingPageLink
+        fields = ['href', 'rel', 'link_type', 'title']
+
+
+class LandingPageSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = LandingPage
+        fields = ['id', 'title', 'description', 'links', 'stac_version']
+
+    id = serializers.CharField(max_length=255, source="name")
+    links = LandingPageLinkSerializer(many=True, read_only=True)
+    stac_version = serializers.SerializerMethodField()
+
+    def get_stac_version(self, obj):
+        return STAC_VERSION
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        api_base = settings.API_BASE
+        request = self.context.get("request")
+        # Add auto links
+        # We use OrderedDict, although it is not necessary, because the default serializer/model for
+        # links already uses OrderedDict, this way we keep consistency between auto link and user
+        # link
+        representation['links'][:0] = [
+            OrderedDict([
+                ('rel', 'self'),
+                ('href', request.build_absolute_uri(f'/{api_base}')),
+                ("type", "application/json"),
+                ("title", "This document"),
+            ]),
+            OrderedDict([
```

Comment:
I like this read-only serializer, but don't see the point of using django-solo then. Or did miss sth?

Reply:
This is not a 100% read only serializer, the `id`, `description` and `title` can be edited. It also have the capability to add other links, like external links for license for example.
I could make it 100% read only and remove the django-solo stuff but then nothing is editable anymore and we loose the flexibility of adding other links. I just thought that having it not 100% static we be a benefit, as we already changed this page twice :-). I let you decide if we keep it as is or make it 100% read only ?

Reply:
Ok, let's keep it for the moment. If we see that it will be more or less stable we can still make it 100% static (or if get troubles with django-solo)

Reply:
:+1: